scriptmodules/emulators/np2pi.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="np2pi"
+rp_module_desc="NEC PC-9801 emulator"
+rp_module_help="ROM Extensions: .d88 .d98 .88d .98d .fdi .xdf .hdm .dup .2hd .tfd .hdi .thd .nhd .hdd"
+rp_module_section="exp"
+
+function depends_np2pi() {
+    local depends=(libsdl1.2-dev libasound2-dev libsdl-ttf2.0-dev ttf-sazanami-gothic)
+    getDepends "${depends[@]}"
+}
+
+function sources_np2pi() {
+    git clone https://github.com/eagle0wl/np2pi.git
+}
+
+function build_np2pi() {
+    cd np2pi/sdl
+    make -j 1 -f makefile.rpi
```

Comment:
Does this only work on the RPI or ? retropie supports other platforms - so the module needs to either be hidden via `!platform` flags or have logic for the build to work on x11 target etc.